logstash-core/lib/logstash/api/modules/base.rb

```diff
@@ -7,15 +7,16 @@ module Modules
       class Base < ::Sinatra::Base
         helpers AppHelpers
 
+        # These options never change
+        # Sinatra isn't good at letting you change internal settings at runtime
+        # which is a requirement. We always propagate errors up and catch them
+        # in a custom rack handler in the RackApp class
         set :environment, :production
+        set :raise_errors, true
+        set :show_exceptions, false
```

Comment:
why not showing exceptions for test? might this help for test I understand, but could we hide some errors also?